gateway/src/node/storage/credential.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2023 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::node::storage::models::PendingStoredCredential;
+
+#[derive(Clone)]
+pub(crate) struct CredentialManager {
+    connection_pool: sqlx::SqlitePool,
```

Comment:
do we really need a separate database for credentials? why not combine it with bandwidth given they serve very similar purpose.

though i could see some possible merit of separating it

Reply:
I didn't think about it, I just blended in existing code